core/animation.cpp

```diff
@@ -7,7 +7,7 @@
  * repository for more details.
  **********************************************************************/
 
-#include <climits>
+#include <limits>
```

Comment:
Couldn't build the project due to an error (missing `std::numeric_limits` in `std` library) - this fixed it!